Formula/poker-emacs.rb

```diff
@@ -21,8 +21,8 @@ def install
       (load "poker")
       (print (poker-random-deck))
     EOS
-    assert_equal 52, shell_output("emacs -Q --batch -l #{testpath}/test.el").
-      strip.gsub(/[()"]/, "").
-      split(" ").length
+    assert_equal 52, shell_output("emacs -Q --batch -l #{testpath}/test.el")
+      .strip.gsub(/[()"]/, "")
```

Comment:
The hound config might be out of date; hold off on changing these until we see what `brew audit` says.